syntax="proto3";

option go_package=".;pb";

message LoginRequest {

  string username = 1;
  string password = 2;
  string email =3;
}


message LoginResponse {
  string username = 1;
  string email = 2;
  string playGameTimes = 3;
  string scores = 4;
}


message MsgInfo {
  string msg = 1;
  string createTime = 2;
  LoginResponse userInfo = 3;
}